jberet-core/src/main/java/org/jberet/repository/JdbcRepository.java

```diff
@@ -146,11 +146,23 @@ public JdbcRepository(final Properties configProperties) {
             this.dbUrl = dbUrl;
             final String dbUser = configProperties.getProperty(DB_USER_KEY);
             if (dbUser != null) {
-                dbProperties.setProperty("user", dbUser.trim());
```

Comment:
jberet.properties configuration only applies to jberet-se, so it will be good to limit this change to jberet-se. For instance, we can contain this changes in `BatchSEEnvironment` right after loading the properties.  This also has the benefit of getting the right values as early as possible.

db config values are also used by mongo job repository.